Tweak.xm

```diff
@@ -2,10 +2,19 @@
 #import <UIKit/UIKit.h>
 #import <libactivator/libactivator.h>
 #import <objc/runtime.h>
-
 #import "Private.h"
 
+#define PREF_PATH @"/var/mobile/Library/Preferences/com.bensge.wipi.plist"
+
 static BOOL shouldShowPicker = NO;
+static BOOL longHoldEnabled;
+
+static void LoadSettings()
```

Comment:
You should follow a casing convention for function names. Maybe `loadSettings()`.